plenum/server/quota_control.py

```diff
@@ -0,0 +1,46 @@
+
+class QuotaControl:
+    def __init__(self,
+                 max_node_message_count_limit,
```

Comment:
Seems like "max" and "limit" are duplicates, I think it would be better to choose one of them (may be "limit" is more appropriate).

Reply:
This is intentional. This class has properties like `node_message_count_limit` which can vary from 0 to defined max values. Now this is binary (either 0 or max), but in future this will most likely change.